newm-server/src/main/kotlin/io/newm/server/database/migration/V52__CreateMarketplace.kt

```diff
@@ -0,0 +1,55 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V52__CreateMarketplace : BaseJavaMigration() {
```

Comment:
I bumped the version I'm working on to 53. Don't want to get burned by that one again.